workflow/rules/eval.smk

```diff
@@ -106,12 +106,28 @@ rule intersect_calls_with_target_regions:
         "<(bcftools view {input.bcf}) -wa -f 1.0 -header > {output}) 2> {log}"
 
 
+rule restrict_to_reference_contigs:
+    input:
+        calls="results/filtered-variants/{callset}.bcf",
+        calls_index="results/filtered-variants/{callset}.bcf.csi",
+        ref_index="resources/reference/genome.fasta.fai",
+    output:
+        "results/filtered-variants/{callset}_restricted.bcf",
+    log:
+        "logs/restrict-to-reference-contigs/{callset}.log",
+    conda:
+        "../envs/bcftools.yaml"
+    shell:
+        "bcftools view --regions $(cut -f1 {input.ref_index} | tr '\\n' ',') {input.calls} | "
+        "bcftools reheader -f {input.ref_index} > {output} 2> {log}"
```

Comment:
```suggestion
        "(bcftools view --regions $(cut -f1 {input.ref_index} | tr '\\n' ',') {input.calls} |"
        " bcftools reheader -f {input.ref_index} > {output}) 2> {log}"
```
The parentheses are needed to ensure that the stderr of both the view and the reheader (not only reheader) is piped into the logfile.